libr/debug/p/debug_native.c

```diff
@@ -1292,7 +1292,6 @@ struct r_debug_plugin_t r_debug_plugin_native = {
 #elif __x86_64__
 	.bits = R_SYS_BITS_32 | R_SYS_BITS_64,
 	.arch = "x86",
-	.canstep = 1,
```

Comment:
Otherwise you define two times the same field of the structure. At line L1297 and L1299 of this file we are already setting the value of this field.